src/cpl.rs

```diff
@@ -125,7 +228,121 @@ impl Clone for CslStringList {
     }
 }
 
+impl Debug for CslStringList {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let mut b = f.debug_tuple("CslStringList");
+
+        for e in self.iter() {
+            b.field(&e.to_string());
+        }
+
+        b.finish()
+    }
+}
+
+impl Display for CslStringList {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        // CSLPrint would be preferable here, but it can only write to a file descriptor.
```

Comment:
Should we use `iter` instead?

Reply:
Thanks. 🤦  I reimplemented that 3 times, reevaluating my assumptions, and lost the forest for the trees.